nixos/modules/services/monitoring/prometheus/exporters.nix

```diff
@@ -301,6 +302,21 @@ in
         Please specify either 'services.prometheus.exporters.sql.configuration' or
           'services.prometheus.exporters.sql.configFile'
       '';
+    } {
+      assertion = cfg.scaphandre.enable -> (pkgs.stdenv.hostPlatform.isx86_64 == true);
+      message = ''
+        Only x86_64 host platform architecture is not supported.
```

Comment:
I think you meant

```suggestion
        Only x86_64 host platform architecture is supported.
```

Reply:
And are you sure the (build) host should be x86_64? Or do you mean the (build) target, i.e: the machine that will run the code?

Reply:
Well I mean the build target indeed.